tools/webpack.config.js

```diff
@@ -167,14 +168,6 @@ const config = {
 
 // Optimize the bundle in release (production) mode
 if (!isDebug) {
-  config.plugins.push(new webpack.optimize.UglifyJsPlugin({
```

Comment:
Is this applied automatically? If yes with what parameters?

Reply:
Yes, that comes with the new `mode: production` in webpack 4... But the params are some defaults set by webpack... We could disable the default minifying of webpack4 though and put an uglifyjsPlugin in ourselves... What do you think?